airflow/jobs/scheduler_job.py

```diff
@@ -668,16 +668,14 @@ def _process_task_instances(self, dag, task_instances_list, session=None):
                     self.log.debug('Queuing task: %s', ti)
                     task_instances_list.append(ti.key)
 
-    def _process_dags(self, dagbag, dags, tis_out):
+    def _process_dags(self, dags, tis_out):
```

Comment:
While at it, can we add type annotations?

Reply:
I prefer not to add type annotations for core classes. We have a problem with cyclical imports.